src/lib/components/events/EventCardGrid.svelte

```diff
@@ -1,17 +1,29 @@
 <script lang="ts">
-	import { Button } from '@emerald-dao/component-library';
 	import Icon from '@iconify/svelte';
 	import type { Event } from '$lib/types/event/event.interface';
+	import { compareDates } from '$lib/utilities/compareDates';
+	import DaysLeft from '../../../routes/admin/events/[id]/_components/atoms/DaysLeft.svelte';
+	import Status from '../../../routes/admin/events/[id]/_components/atoms/Status.svelte';
```

Comment:
@IgnacioDebat suggestion here:
Shouldn't these two components (Status and DaysLeft) be in the `src/lib/components/events` directory? 
That way we have everything related to events UI in the same directory..